spec/api/entities/ccr/defendant_spec.rb

```diff
@@ -25,12 +25,12 @@
 
   it 'returns main defendant true for the defendant created first' do
     @created_at = 1.minute.ago
-    expect(response[:main_defendant]).to eql true
+    expect(response[:main_defendant]).to be_truthy
   end
 
   it 'returns main defendant false for defendant created after other defendants' do
     @created_at = 1.minute.from_now
-    expect(response[:main_defendant]).to eql false
+    expect(response[:main_defendant]).to be_falsey
```

Comment:
Was there a reason for preferring .to be_falsey over .to be false when replacing eql false? (and .to be_truthy over .to be true)? (is it possible to chose what rubocop autocorrects to?)

Reply:
Rubocop changes `eql false` to `be false`, which will check that the value is exactly `false`. `be_falsey` checks that the value evaluates to `false`, so `nil` will also be accepted.

In most places I had been changing the `eql` to `eq`, which is slightly different but I think (maybe I'm wrong) this is what is actually intended. `eql` compares as `actual.eql? expected`, which requires that `actual` and `expected` have the same type as well as the same value, whereas `eq` compares as `actual == expected`, where type conversion is allowed. I believe that for `true` and `false` this makes `eq false` identical to `be_falsey`.

What do people think? Should we be checking for exactly boolean values in some cases or is it generally OK to check `truthy` and `falsey`?

Reply:
As per other comment it is spec'ing this method 
```
  def main_defendant
    object.claim.defendants.unscope(:order).order(created_at: :asc)&.first == object || false
  end
  ```
so without investigating impact further i would say `be false` , `be true` should be used